ff4j-spring-boot-autoconfigure/pom.xml

```diff
@@ -59,10 +59,12 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
```

Comment:
This is [already in test scope](https://github.com/ff4j/ff4j-spring-boot-starter-parent/blob/master/pom.xml#L181)